src/v2/Apps/Partner/Routes/Works/index.tsx

```diff
@@ -24,6 +26,17 @@ export const Artworks: React.FC<PartnerArtworkFilterProps> = ({
 }) => {
   const { sidebar } = partner
 
+  const savedSearchEntity: SavedSearchEntity = {
+    placeholder: "Partner Page", // TODO: Clarify and update it later
+    defaultArtists: [],
+    owner: {
+      type: OwnerType.partner,
+      slug: partner.slug,
+      id: partner.internalID,
+      name: partner.name ?? "",
+    },
+  }
+
```

Comment:
**minor, non-blocking:** suggest removing this and moving it to a separate PR since it's not necessarily related to rendering the filter pills (see my other comment about making those separate concerns)